flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/InputChannel.java

```diff
@@ -74,6 +75,8 @@
     /** The current backoff (in ms). */
     private int currentBackoff;
 
+    protected int currentBufferSize;
```

Comment:
I think changes from this file should belong to the following commit

Reply:
On the second thought, you are storing the `currentBufferSize` only for the testing purposes? If so, couldn't you add this code to `TestInputChannel` instead of modifying the production code?

Reply:
it makes sense. I have rewritten it